examples/SampleApp/android/app/google-services.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
we probably shouldn't put this in there. we should put a temporary example and share the real one internally and have that substituted in/out during the build